### PR TITLE
[DOCS] Update text to include name for the new transform in the transform troubleshooting docs

### PR DESCRIPTION
**Pull request by joepeeples**
Addresses #1224.

Preview: [Troubleshooting | Management tools](https://security-docs_1228.docs-preview.app.elstc.co/guide/en/security/master/ts-management.html)

### CODE REVIEW

**Review by nastasha-solomon — APPROVED** [review]
LGTM!

**Review by kevinlog — APPROVED** [review]
LGTM!  Thanks for getting this in!

**Review by benironside — APPROVED** [review]
LGTM

File: docs/troubleshooting/management/ts-management.asciidoc [hunk 1]
```diff
@@ -20,8 +20,8 @@ image::images/endpoints-transform-failed.png[Endpoints page with Required transf
 To restart a transform that’s not running:
 
 . Go to *Kibana* -> *Stack Management* -> *Data* -> *Transforms*.
-. Enter `endpoint.metadata_current-default` in the search box to find the transform for {endpoint-sec}.
-. Click the *Actions* icon (*...*) and do one of the following, depending on the value in the *Status* column:
+. Enter `endpoint.metadata` in the search box to find the transforms for {endpoint-sec}.
+. Click the *Actions* icon (*...*) and do one of the following for each transform, depending on the value in the *Status* column:
```
Comment (jmikell821) [inline docs/troubleshooting/management/ts-management.asciidoc:24]:
```suggestion
. Click the *Actions* button (*...*) and do one of the following for each transform, depending on the value in the *Status* column:
```

**Review by jmikell821 — APPROVED** [review]
One small nit then this is good to go.

**Pull request merged**